### PR TITLE
remove automatic_heating_enabled, automatic_cooling_enabled

### PR DESCRIPTION
**Pull request by kainpets**

### CODE REVIEW

File: docs/api-clients/thermostats/update-a-thermostat.md [hunk 1]
```diff
@@ -30,8 +30,6 @@ The primary attribute you can modify is the [default climate setting](../../prod
 seam.thermostats.update(
   device_id: "a83690b2-2b70-409a-9a94-426699b84c97",
   default_climate_setting: {
-    "automatic_cooling_enabled": true,
-    "automatic_heating_enabled": true,
     "cooling_set_point_fahrenheit": 70
```
Comment (phpnode) [inline docs/api-clients/thermostats/update-a-thermostat.md:33]:
```suggestion
    "hvac_mode_setting": "heat_cool",
    "cooling_set_point_fahrenheit": 70
```

File: docs/products/thermostats/retrieving-thermostats.md [hunk 2]
```diff
@@ -25,8 +25,6 @@ seam.thermostats.list()
     properties={
         'available_hvac_mode_settings': ['off', 'cool', 'heat', 'heat_cool'],
         'current_climate_setting': {
-            'automatic_cooling_enabled': True,
-            'automatic_heating_enabled': True,
             'cooling_set_point_celsius': 25,
```
Comment (phpnode) [inline docs/products/thermostats/retrieving-thermostats.md:28]:
```suggestion
            'hvac_mode_setting': 'heat_cool',
            'cooling_set_point_celsius': 25,
```

**Review by phpnode — APPROVED** [review]
we just need to set the `hvac_mode_setting` in a couple of places

File: docs/api-clients/thermostats/update-a-thermostat.md (hunk 1 above)
Comment (kainpets) [reply docs/api-clients/thermostats/update-a-thermostat.md:33]: I can put it in but this whole file gets deleted in https://github.com/seamapi/docs/pull/309

Comment (phpnode) [reply docs/api-clients/thermostats/update-a-thermostat.md:34]: let's put it back in for now

**Pull request merged**